### PR TITLE
Fix wrong context when adding qualifier/reference to new statement

### PR DESCRIPTION
**Pull request by thiemowmde**
Note that this is not tested! This change may have side effects.

[Bug: T102324](https://phabricator.wikimedia.org/T102324)

### CODE REVIEW

File: modules/ext.PropertySuggester.EntitySelector.js [hunk 1]
```diff
@@ -140,12 +140,12 @@
...
 		_getPropertyContext: function() {
-			if ( this._isInNewStatementView() ) {
-				return 'item';
-			} else if ( this._isQualifier() ) {
+			if ( this._isQualifier() ) {
 				return 'qualifier';
 			} else if ( this._isReference() ) {
 				return 'reference';
+			} else if ( this._isInNewStatementView() ) {
+				return 'item';
```
Comment (JeroenDeDauw) [inline modules/ext.PropertySuggester.EntitySelector.js]: Why an if-elseif-else contraption >_> djeez

Comment (thiemowmde) [inline modules/ext.PropertySuggester.EntitySelector.js]: I'm not sure how this code could be made better. Do you have a suggestion?

Comment (JeroenDeDauw) [inline modules/ext.PropertySuggester.EntitySelector.js]: Guards

Comment (mariushoch) [inline modules/ext.PropertySuggester.EntitySelector.js]: I don't see how guards might be useful here. Maybe a `switch ( null )` would be more appropriate here.

**Comment by mariushoch** [comment]
This leads to invalid API calls (`Exception Caught: provide either entity-id parameter 'entity' or a list of properties 'properties'`).

**Comment by thiemowmde** [comment]
This is trivial, and not a working solution. Easier to start over.

**Pull request closed without merging**